sfmc_release_notes/SF_managed_clusters_ga_release_notes.md

```diff
@@ -0,0 +1,40 @@
+# Microsoft Azure Service Fabric managed clusters general availability release
+
+Azure Service Fabric managed clusters are now generally available. We first introduced Service Fabric managed clusters in October, 2020. Service Fabric managed clusters enable customers to deploy Service Fabric clusters as a single ARM resource, and reduce the complexity associated with common cluster operations such as scaling, and certificate rollovers. In this release we are bringing additional capabilities such as availability zone spanning, stateless node types, application secrets, application managed identities, and the ability to configure cluster upgrades.
```

Comment:
Not sure I like marketing speak in release notes. I'd prefer more to the point like this? thoughts?

Azure Service Fabric managed clusters are now generally available. In this release we are bringing additional capabilities such as availability zone spanning, stateless node types, application secrets, application managed identities, and the ability to configure cluster upgrades.

idea, pull in/up the region availability statement?

Reply:
I agree this sounds better. I also moved up the region statement.